demo/owner/README.md

```diff
@@ -180,4 +180,33 @@ The PKCS12 keystore file \<fido-iot-src\>/demo/owner/owner_keystore.p12 contains
 
 # Troubleshooting
 
-As the H2 DB grows, larger heap space will be required by the application to run the service. Default configured heap size is `256 MB`. Increase the heap size appropriately in `demo/owner/owner-entrypoint.sh` to avoid heap size issues.
\ No newline at end of file
+As the H2 DB grows, larger heap space will be required by the application to run the service. Default configured heap size is `256 MB`. Increase the heap size appropriately in `demo/owner/owner-entrypoint.sh` to avoid heap size issues.
+
+# Configuring Owner for HTTPS/TLS Communication
+
+By default, the Owner uses HTTP for all communications on port 8042. In addition to that, the Owner can be configured to handle HTTPS request from the device.
+
+- Generate the Keystore/Certificate for the Owner
+
+  * Ensure that the web certificate is issued to the resolvable domain of the Owner server.
+
+- Copy the generated Keystore/Certificate to `demo/owner/certs` folder.
+
+- Copy the truststore containing all the required certificates to `demo/owner/certs` folder.
+
+- Update the following environment varibles in `demo/owner/owner.env` file
+
+    |  Variable              |  Value            |             Description       |
+    | -----------------------|-------------------|-------------------------------|
+    | owner_protocol_scheme  | https             | To enable HTTPS communication.|
+    | owner_https_port       | port number       | The given port will be used for HTTPS communication. |
+    | fido_ssl_mode          | TEST / PROD       | If set to `TEST`, then SSL verification is disabled. If set to `PROD`, then certificate verification is initiated. |
+    | owner_ssl_keystore     | keystore-filename | Filename of Keystore that is present in the certs folder.|
+    | owner_ssl_keystore-password| keystore-password | Password of the keystore. |
+    | ssl_truststore         | truststore-filename  | Filename of truststore that is present in the certs folder. Not required in `TEST` mode. |
+    | ssl_truststore_password| truststore-password | Password of the truststore. Not required in `TEST` mode. |
+    | ssl_truststore_type    | truststore-type   | Type of truststore. eg: JKS ,PKCS12   |
+    | owner_to0_rv_blob      | to0_rv_blob       | Contains the to0_rv_blob used by device to connect with the Owner during T02. Eg: https://localhost:<owner-https-port>?ipaddress=127.0.0.1 |
```

Comment:
"\<owner-https-port\>" is not rendering. Add "\\" before the opening and ending angular braces.

Reply:
Done.